app-office/planner/planner-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Project manager for Gnome"
+HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/planner.git"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64"
+fi
+
+SLOT="0"
+LICENSE="GPL-2+"
+IUSE="database-gda examples gtk-doc" # eds
+
+# deps on bottom from qa-vdb
```

Comment:
I don't think these should be added like that when it just says something. All of them are just pulled in automatically by gtk+ (though surprised about at-spi2-core) and we don't really list them in other packages like this, unless upstream build system explicitly checks for them.

Reply:
saw that in some gnome packages before, thats why I added them like his. Will remove them if you don't like it though